.maint/contributors.json

```
@@ -130,6 +130,11 @@
     "name": "Naveau, Mikaël",
     "orcid": "0000-0001-6948-9068"
   },
+  {
+    "affiliation": "Department of Radiology, University Hospital of Lausanne",
```

This is the correct affiliation as recommended by CHUV:
```suggestion
    "affiliation": "Department of Radiology, Lausanne University Hospital and University of Lausanne",
```